src/main/java/core/basesyntax/model/OperationFruitDto.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.model;
+
+import core.basesyntax.operationstrategy.OperationService;
+
+public class OperationFruitDto {
+    private OperationService operationService;
```

Comment:
we cannot make the handler the field of the dto - please refactor this
create a map with operation as key and handler as value in main() and use it to choose the handler

Reply:
I have a map with operation in class OperationStrategy - Is this correct?